lib/menu.ts

```diff
@@ -113,27 +121,62 @@ export class Menu extends Disposable implements IPopupContent {
     );
     this.onDispose(() => domDispose(this.content));
 
+    if (options.startIndex !== undefined) {
+      // Cannot check which elements are visible (have an offset height) on creation since none
+      // of the items are attached to the dom yet.
```

Comment:
Maybe clarify with "Not using isSelectable because..."? (It seems that's the purpose of the comment)